modules/migration-mtc-release-notes-1-6.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+
+[id="migration-mtc-release-notes-1-6_{context}"]
+= {mtc-full} 1.6 release notes
+
+The release notes for {mtc-full} ({mtc-short}) version 1.6 describe new features and enhancements, deprecated features, and known issues.
+
+[id="new-features-and-enhancements-1-6_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* State migration
++
+You can migrate an application's state by selecting persistent volume claims (PVCs) and Kubernetes objects for migration.
```

Comment:
Normal migrations have always migrated state. The way this is worded suggests this is a new feature to me. I think I would stress, with this release we've added new functionality that allows for ad-hoc, repeatable state only transfers on a PVC by PVC basis.